README.md

```diff
@@ -82,6 +81,11 @@ Django is server-side python framework that allows developers to create web apps
 1. [Django Packages](https://djangopackages.org/) contains different django packages for almost all different categories.
   You can compare different packages in same category and choose your favorite one.
 
+#### Unit Tests
+
+[The Django Test Framework](https://github.com/anant-joshi/resources/wiki/The-Django-Test-Framework)  
+@anant-joshi's guide to the Django Test Framework
```

Comment:
Do we really need your name considering the moment someone clicks on it, they're redirected to your repo? :stuck_out_tongue: